apps/portals-base/package.json

```diff
@@ -33,26 +32,12 @@
   "scripts": {
     "clean": "rimraf coverage",
     "lint": "eslint src",
-    "copy-assets": "cp src/config/style/favicon* public/; cp src/config/style/logo.svg public/; cp src/config/style/socialmedia.png public/",
-    "save-build-date": "date > public/build-date.txt",
-    "start": "vite",
```

Comment:
No need for a `start` script for this package anymore, it's just portals-related code.